app/guid-file/-components/file-detail-layout/styles.scss

```diff
@@ -0,0 +1,56 @@
+.Container {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+}
+
+.MainColumn {
+    flex-grow: 3;
+}
+
+.RightColumn {
+    flex-grow: 1;
```

Comment:
Right now, for smaller desktop sizes this column gets a bit squished and leads to some text wrapping, so adding a `min-width:300px` or something like that should prevent some of those issues as we develop the tags and revisions widgets.